magento-research/pwa-studio
# Prevent Rest api call on minicart sidebar if cart is empty

## Description

I've prevented Rest API call if the cart is empty.

## Related Issue





Closes #1104.

## Verification Steps

1. Go to the pwa-studio storefront.
2. Open developer console and go to the network tab.
3. Click on the mini-cart icon at the top right section.
4. See you will find rest API call even cart is empty.

## Screenshots / Screen Captures (if appropriate):
**Before Fixed:** 
![2019_04_02_Selection_002](https://user-images.githubusercontent.com/38535982/55397047-ec3baa80-5562-11e9-9b3f-cd83a3b6ec6c.png)

**After Fixed:**
![2019_04_02_Selection_001](https://user-images.githubusercontent.com/38535982/55397066-f52c7c00-5562-11e9-9036-01f4a9b8365d.png)

## Proposed Labels for Change Type/Package




## Checklist:
